client_app/src/components/shelter/ShiftsModal.js

```diff
@@ -45,25 +46,33 @@ export const ShiftsModal = props => {
             Signed-Up Volunteers
           </h3>
         </span>
-        <p>{"Date: " + volunteerShifts.shift.date}</p>
-        <p>{"Shift: " + volunteerShifts.shift.name ?
-          volunteerShifts.shift.name + ": " + volunteerShifts.shift.startTime + " - " + volunteerShifts.shift.endTime 
-          : volunteerShifts.shift.startTime + " - " + volunteerShifts.shift.endTime}</p>
-        <p>{"Current Volunteer Count: " + volunteerShifts.shift.currentVolunteers}</p>
-        <p>{"Required Volunteer Count: " + volunteerShifts.shift.requiredVolunteers}</p>
-        <table className="shiftsTable">
-          <thead className="shiftsTableHeader">
-            <tr>
-              <th>Name</th>
-              <th>Phone Number</th>
-              <th>Email</th>
-              <th>Send E-mail</th>
-            </tr>
-          </thead>
-          <tbody className="shiftsTableBody">
-            {renderShifts()}
-          </tbody>
```

Comment:
I think the way you have this set up and rendering is fine, it cant be any smaller as we discussed in class, because it wouldn't be easy to read.